Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
    fid = stack.enter_context(open(os.fspath(file), "rb"))
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    fid = stack.enter_context(open(os.fspath(file), "rb"))
    fid = stack.enter_context(open(os.fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
Traceback (most recent call last):
  File "/scratch/ed2189/JMP/GMM_quad_old/job-main.py", line 33, in <module>
    quadratic_j_j_k = np.load('GMM_quad_old/data/quadratic_characteristic_j_j_k.npy')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ext3/miniforge3/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py", line 451, in load
    fid = stack.enter_context(open(os.fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'GMM_quad_old/data/quadratic_characteristic_j_j_k.npy'
srun: error: cs257: tasks 15-29: Exited with exit code 1
srun: Terminating StepId=60119191.0
slurmstepd: error: *** STEP 60119191.0 ON cs140 CANCELLED AT 2025-05-05T16:35:01 ***
srun: error: cs473: tasks 105-119: Exited with exit code 1
srun: error: cs340: tasks 60-74: Exited with exit code 1
srun: error: cs477: tasks 135-149: Exited with exit code 1
srun: error: cs332: tasks 45-59: Exited with exit code 1
srun: error: cs423: tasks 75-89: Exited with exit code 1
srun: error: cs474: tasks 120-134: Exited with exit code 1
srun: error: cs140: tasks 0-14: Exited with exit code 1
srun: error: cs468: tasks 90-104: Exited with exit code 1
srun: error: cs322: tasks 30-44: Exited with exit code 1
